test/fixtures/.flutterflowignore

```diff
@@ -0,0 +1,8 @@
+file
+dir/*
```

Comment:
When I add `dir/` to `.gitignore` it ignores the entire directory, but this implementation doesn't. Let's update it to support ignoring directories?

Reply:
Done, also added support for comments starting with `#` like in git.